src/policy/semantic.rs

```diff
@@ -499,6 +499,34 @@ impl<Pk: MiniscriptKey> Policy<Pk> {
         ret
     }
 
+    /// Helper function for recursion in `absolute timelocks`
+    pub fn real_absolute_timelocks(&self) -> Vec<u32> {
```

Comment:
nit: if this is a helper function, we'd rather not make it public?

Reply:
Ideally, this should be been fixed in cac78151935e83a0cd0eb58da0cca175e8e4ba4e and not in the next commit, but it' okay :)